### DISCUSSION TITLE
Plugin list

### DISCUSSION THREAD

**Discussion by jrhender** [decentralized-identity/veramo#691] [Q&A] [answered]
Veramo use a plugin architecture and the "Plugins" [documentation page](https://veramo.io/docs/veramo_agent/plugins) mention the vision of an ecosystem of highly interoperable functions. Is there a registry or a list of plugins that are available (in addition to the core plugins) thus far in the ecosystem?

**Comment by mirceanis** [comment, marked as answer]
The short answer: no, there is no list (yet).

Up to this date we are not aware of many public plugins.
We are aware of a few closed-source plugins for key management, and we have experimented in the past with plugins for different DID methods, credentialStatus and different other protocols and UI interactions.
The plugins you see embedded in this repository (all the packages except for `core` and `cli`) are so far the only plugins that are officially maintained and known to work with the latest veramo.

There is a public [.well-known DID config plugin](https://github.com/SertoID/veramo-plugin-did-config) from our sister team [Serto.id](https://www.serto.id/).

P.S. If you're reading this answer and think your plugin should be mentioned, please use the [Show & Tell](https://github.com/uport-project/veramo/discussions/categories/show-and-tell) section to show off your work.

**Comment by jrhender** [comment]
Sphereon seems to have made some plugins that extend Veramo: https://github.com/Sphereon-Opensource/ssi-sdk
